src/funcs.cpp

```diff
@@ -0,0 +1,24 @@
+#include "funcs.hpp"
+
+
+/* Functions */
+void progresssBar(double progress, std::string msg)
```

Comment:
This should be turned off for now to avoid compatibility issues

Reply:
```suggestion
// void progresssBar(double progress, std::string msg)
```
This change is live in the newest commit.